Rearrange and improve documentation (#53)
epics-training/training-vm: doc/instructions.md

# Setting Up a Training-VM Instance

The Training-VM is available in four flavours in their stable/current versions:
- rocky: [RockyLinux](https://rockylinux.org/) (compatible to RHEL)
- fedora: [Fedora Linux](https://fedoraproject.org/)
- debian: the "stable" release of [Debian Linux](https://www.debian.org/)
- ubuntu: [Ubuntu Linux](https://ubuntu.com/)

Not all modules are working on all flavours.

Not all modules are working on Mac using Apple Silicon processors.

## Create the Virtual Machine

The first step is to create a VM in the desired flavour.

There are two ways to do that:

> 1. You can create the VM in the traditional manual way

Reviewer: ```suggestion
1. You can create the VM step-by-step from scratch
```

Mentioning "traditional" once  (in the next sentence) is probably enough ;)
